### PR TITLE
Footer

### LINKED ISSUE #115: Fix : Minimize Footer Size
## Summary

Make it simple, remove unnecessary things, and keep the usable ones.

### Screenshots
N/A

### Basic example

N/A

### Motivation

To have a functional and simple footer

### PR DESCRIPTION
**Pull request by sarahaboudalal**

### CODE REVIEW

**Review by ayubabdullah — COMMENTED** [review]
Doesn't it look better if the content be center on small screen ?

**Comment by OmarBassatni97** [comment]
I prefer the on before, this one feels empty

**Review by OmarBassatni97 — APPROVED** [review]
Way better, Bravo Sara

**Review by ayubabdullah — APPROVED** [review]
Great work sara, just align the text "designed for your needs" center and check if it's look better or not

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -1,249 +1,75 @@
...
+            src="https://cdn.discordapp.com/attachments/1031834305703460906/1035627738440159303/Asset_23.png"
+            className="mr-3 h-32 w-min lg:self-start self-center"
+            alt="Flowbite Logo"
+          />
+          <span className="self-center text-3xl lg:mb-0 mb-3 font-semibold text-primary">
+          Designed to meet your needs.
+          </span>
+          </Link>
+        <div className="col-span-1 flex lg:my-0 my-3 mx-3 lg:justify-start justify-center text-center">
+        <ul className="mb-6 text-lg text-primary sm:mb-0 font-base justify-self-center">
```
Comment (MARWAHAKMI) [inline src/components/Footer/Footer.jsx:22]: If you can make links aligned left on large screens and centered on small screens, it would be better.

**Review by MARWAHAKMI — APPROVED** [review]
LGTM 👌
but I prefer Navbar and Footer have the same bg color, but it looks good
Well Done

**Review by Iseluin — APPROVED** [review]

**Pull request merged**